tests/Integration/SpannerDatabaseInstanceManager.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Oat\DbalSpanner\Tests\Integration;
+
+use Google\Cloud\Core\Exception\GoogleException;
+use Google\Cloud\Spanner\Database;
+use Google\Cloud\Spanner\Instance;
+use Google\Cloud\Spanner\SpannerClient;
+use LogicException;
+
+class SpannerDatabaseInstanceManager
+{
+    private const KEY_FILE_ENV_VARIABLE = 'GOOGLE_APPLICATION_CREDENTIALS';
+    private const CONFIGURATION_NAME = 'regional-europe-west1';
+
+    /** @var string */
+    private $projectName;
+
+    /** @var SpannerClient */
+    private $client;
+
+    public function __construct()
+    {
+        $keyFile = json_decode(file_get_contents(getenv(self::KEY_FILE_ENV_VARIABLE)), true);
+        $this->projectName = $keyFile['project_id'];
+
+        try {
+            $this->client = new SpannerClient(['keyFile' => $keyFile]);
+        } catch (GoogleException $exception) {
+            throw new LogicException('gRPC extension is not installed or enabled.');
```

Comment:
It is the only one mentioned in the phpdoc of Google\Cloud\Spanner\SpannerClient::__construct... so I took it for granted since the documentation quality is generally high.

Reply:
ok, good , thx :)